governance/src/main/java/org/apache/servicecomb/governance/policy/AbstractPolicy.java

```diff
@@ -27,4 +27,27 @@ public boolean isValid() {
     }
     return true;
   }
+
+  public String getTimeDuration(String time, int defaultValue) {
```

Comment:
1. add a method `Duration stringLiteratalToDuration(String time, Duration defaultValue)`
2. catch exception if time is not a valid time.

Reply:
done